src/Spago/Messages.hs

```diff
@@ -242,3 +242,5 @@ sourceImportsTransitiveDependency transitive = makeMessage $
   [ "Some of your project files import modules from packages that are not in the direct dependencies of your project."
   , "To fix this error add the following packages to the list of dependencies in your config:"
```

Comment:
Comment related to the `unusedDependency` block above (must expand due to this github feature gap https://github.com/isaacs/github/issues/284):

It would be nice if we could also offer the same convenience for unused dependencies here with a `spago uninstall` or `spago remove` line. I recall `uninstall`/`remove` being proposed before, but I can't find the issue for it.

Reply:
IIRC it was a thread on slack. I don't think providing a command here is as urgent as with the other error, as this is just a warning and won't fail your build